### ISSUE TITLE
"Segmentation fault" on Amazon deep learning AMI, gpu-compute instance

### ISSUE THREAD

**Issue by ben-arnao** [tensorflow/tensorflow#37513]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information** 
- Have I written custom code (as opposed to using a stock
example script provided in TensorFlow): No
- OS Platform and Distribution (e.g.,
Linux Ubuntu 16.04):  Deep Learning AMI (Amazon Linux) Version 27.0 (ami-0e2a8509db267f072)
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if
the issue happens on mobile device: 
- TensorFlow installed from (source or
binary): - TensorFlow version (use command below): source activate tensorflow2_p36 on AMI
- Python version: - Bazel 3.6
version (if compiling from source):
- GCC/Compiler version (if compiling from
source):  None
- CUDA/cuDNN version: - GPU model and memory:
p3.2xlarge AWS instance.  NVIDIA Tesla V100 15gb. 10.1 cuda, 7.6 cudnn

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**

model.fit() just stops randomly with "segmentation fault" no other info

**Describe the expected behavior**

No errors or at least more debugging info

**Standalone code to reproduce the issue** 
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

**Comment by ravikyram** [comment]
@ben-arnao 

Request you to share simple standalone code to reproduce the issue in our environment. It helps us in localizing the issue faster. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ravikyram** [comment]
@ben-arnao 

Any update on this issue please. Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
